ui/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "scality-ui-template",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@fortawesome/fontawesome-free": "^5.7.2",
+    "@kubernetes/client-node": "git://github.com/scality/kubernetes-client-javascript.git#browser",
```

Comment:
I guess at some point we'll want to create a tag in that repo, then point to the tag.